SCClassLibrary/Common/Math/SimpleNumber.sc

```diff
@@ -160,6 +160,26 @@ SimpleNumber : Number {
 		}
 	}
 
+	snap { arg resolution = 1.0, tolerance = 0.05, strength = 1.0;
+		var round = round(this, resolution);
+		var diff = round - this;
+		if (abs(diff) < tolerance) {
+			^this + (strength * diff)
+		}{
```

Comment:
i know this is copy-pasted, but please add a space between brackets here and in line 179 as well.

Reply:
you could change the original as well if you want